src/py/flwr/common/address.py

```diff
@@ -22,14 +22,14 @@
 IPV6: int = 6
 
 DOMAIN_PATTERN: Pattern[str] = re.compile(
-    r"^(localhost)|(([a-zA-Z]{1})|([a-zA-Z]{1}[a-zA-Z]{1})|"
+    r"^(https?:\/\/)?(localhost)|(([a-zA-Z]{1})|([a-zA-Z]{1}[a-zA-Z]{1})|"
```

Comment:
```suggestion
    r"^(https?:\/\/)?(localhost)|(([a-zA-Z]{1})|([a-zA-Z]{1}[a-zA-Z]{1}))"
```

 think there is a closing `)` missing.

Reply:
@tanertopal Nope, the closing bracket is on the 3rd line

Reply:
my bad. Did not realize this is a multi line string.